src/rust/demikernel/libos/mod.rs

```diff
@@ -344,7 +338,6 @@ impl LibOS {
 
     pub fn sgaalloc(&mut self, size: usize) -> Result<demi_sgarray_t, Fail> {
         let result: Result<demi_sgarray_t, Fail> = {
-            timer!("demikernel::sgaalloc");
```

Comment:
we might want to keep these just in case somehow this number goes up (ie., it becomes much more expensive to allocate memory through Demikernel)

Reply:
Done